lib/services/local/access_list.go

```diff
@@ -36,6 +37,8 @@ const (
 
 	accessListMemberPrefix      = "access_list_member"
 	accessListMemberMaxPageSize = 200
+
+	accessListLockTTL = 5 * time.Second
```

Comment:
Can you add a comment on why this lock is needed? Mainly mention that it's needed to prevent a race condition between the access list and the members.